### DISCUSSION TITLE
Quick Astra API tutorial

### DISCUSSION THREAD

**Discussion by synedra** [EddieHubCommunity/support#1408] [Show and tell]
Hi, all, I'm a Developer Advocate at DataStax. Our team is whipping up content to help developers (all free of course). If you want to understand the Astra/Stargate APIs, there's a really quick walkthrough at https://www.katacoda.com/datastax/scenarios/httpie-astra. Lots of other scenarios are getting put together all the time :slight_smile:

**Comment by eddiejaoude** [comment]
Great! This is really useful, thanks 👍

**Comment by mariamar111** [comment]
thanks its useful
